lib/Request.js

```diff
@@ -140,7 +140,7 @@ class Request extends CallableInstance {
       flowType: 'exit'
     })
 
-    this._emitter.emit('sent', event)
+    this._remit._emitter.emit('sent', event)
```

Comment:
As above, this should be using `this._emitter` as it's only relevant to _this_ request object.